src/open_inwoner/components/templates/components/DenhaagAction/DenhaagAction.html

```diff
@@ -0,0 +1,15 @@
+{% load l10n i18n %}
+
+<a href="{{link}}" class="denhaag-action denhaag-action--single">
+    <div class="denhaag-action__content">{{title}}</div>
+    <div class="denhaag-action__details">
+        <div class="denhaag-action__date">
+            <time datetime="2023-09-28T19:47:36.593Z">{{date}}</time>
+        </div>
+        <div class="denhaag-action__actions">
+            <svg width="1em" height="1em" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg" class="denhaag-action__link-icon" focusable="false" aria-hidden="true" shape-rendering="auto">
+                <path d="M12.293 5.293a1 1 0 011.414 0l6 6a1 1 0 010 1.414l-6 6a1 1 0 01-1.414-1.414L16.586 13H5a1 1 0 110-2h11.586l-4.293-4.293a1 1 0 010-1.414z" fill="currentColor"></path>
+            </svg>
+        </div>
+    </div>
+</a>
```

Comment:
Github shows a red no-newline warning here (configure your editor to add it on save).